setup.py

```diff
@@ -937,9 +946,11 @@ def print_box(msg):
     print('-' * (size + 2))
 
 if __name__ == '__main__':
-    # Parse the command line and check the arguments
-    # before we proceed with building deps and setup
+    # Parse the command line and check the arguments before we proceed with
+    # building deps and setup. We need to set values so `--help` works.
     dist = Distribution()
+    dist.script_name = os.path.basename(sys.argv[0])
+    dist.script_args = sys.argv[1:]
```

Comment:
This is so `python setup.py bdist_wheel --help` will work. Usually the call to `setup()` instantiates the `Distribution` and sets the `dist.script_name` and `dist.script_args`, but in this case `dist` is instantiated here.